package.json

```diff
@@ -130,6 +130,6 @@
     ]
   },
   "engines": {
-    "node": ">= 10"
+    "node": ">= 10.21.0"
```

Comment:
If a user has the `engine-strict` flag set to true, they won't be able to use ts-jest with node versions < 10.21.0, right?
Do we want that behavior?

Reply:
this change doesn't impact to end users unless we distribute `ts-jest` with `engineStrict: true` in our `package.json`. I've tested the scenario that end users set `engineStrict: true` but it doesn't prevent from installing `ts-jest`. Only when we explicitly set in our `package.json`, then end users won't be able to install.

So this change is safe, only for local `ts-jest` development.